### ISSUE TITLE
Spamming rooms with players

### ISSUE THREAD

**Issue by Wolf72** [haxball/haxball-issues#475]
![screenshot_19](https://user-images.githubusercontent.com/39432419/43365427-29aefbe0-932d-11e8-82ad-4396995f8058.png)

basro please provide us with ip address or do something about this. (limit 1-2 players per ip that can join same room).

**Comment by olegmisar** [comment]
You can use haxball firewall which 
- protects against packet flooders(probably it is implemented in haxball natively)
- bans players who try to join the same room several times (ban isn't permanent, maybe 10-30 minutes, idk)

You can find this firewall for Linux or Windows on GitHub.

**Comment by Wolf72** [comment]
I blocked his ip with iptables although I don't even know if it works.
I don't think joining same room counts as packet flooding.
Also I can't just ban same player. I don't know if it's same player. (he changes nicknames) And I have automatic unbans every hour which clears all bans. I cannot exclude him from that clear.

**Comment by andriyleu** [comment]
Yeah, I dont understand either why the only method available to unban people unbans everyone.

**Comment by kamild706** [comment]
@Wolf72 How did you get his IP? Is there any better way than looking at network traffic in Wireshark and guessing which address belongs to each user?

**Comment by Leo-MapCreator** [comment]
Get his "conn", and only allow a max of 1 or 2 players from the same conn
